packages/material-ui/src/Grid/Grid.test.js

```diff
@@ -24,68 +30,69 @@ describe('<Grid />', () => {
 
   describe('prop: container', () => {
     it('should apply the container class', () => {
-      const wrapper = shallow(<Grid container />);
-      expect(wrapper.hasClass(classes.container)).to.equal(true);
+      const { container } = render(<Grid container />);
+
+      expect(container.firstChild).to.have.class(classes.container);
     });
   });
 
   describe('prop: item', () => {
     it('should apply the item class', () => {
-      const wrapper = shallow(<Grid item />);
-      expect(wrapper.hasClass(classes.item)).to.equal(true);
+      const { container } = render(<Grid item />);
+
+      expect(container.firstChild).to.not.equal(null);
```

Comment:
shouldn't we test the class name?

Reply:
@oliviertassinari Sure! I don't why I've removed all class assertions. I guess I was overworked :D Thanks for the review :)